youtube_dl/extractor/dplay.py

```diff
@@ -5,40 +5,70 @@
 
 from .common import InfoExtractor
 from ..utils import int_or_none
-
+from ..compat import compat_urlparse
 
 class DPlayIE(InfoExtractor):
-    _VALID_URL = r'http://www\.dplay\.se/[^/]+/(?P<id>[^/?#]+)'
-
-    _TEST = {
-        'url': 'http://www.dplay.se/nugammalt-77-handelser-som-format-sverige/season-1-svensken-lar-sig-njuta-av-livet/',
```

Comment:
The _TEST variable has been changed to an array and two additional tests have been added?  The documentation I read mentioned to do that in the instance of more than one test case.